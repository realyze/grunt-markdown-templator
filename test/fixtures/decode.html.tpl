<% _.forEach(children, function(section) { %>
  <%= section.body %>
</section><%
}) %>
