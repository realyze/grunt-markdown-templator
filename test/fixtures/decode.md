# Solutions {foobar}

This is the <span>Solutions</span> noodle.
