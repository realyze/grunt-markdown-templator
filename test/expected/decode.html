<p>This is the <span>Solutions</span> noodle.</p>
</section>
